### PR TITLE
Add Floating Action Buttons

### PR DESCRIPTION
**Pull request by MaximeJineMaraval**
## Changes description 🧑‍💻

Add new fresh component : FloatingActionButton
Also, fix some readme mistakes and some colors (validated with @Sabrinavigil)

## Context 🤔

Closes https://github.com/Decathlon/vitamin-android/issues/13

## Checklist ✅

 - [x] I have reviewed the submitted code
 - [x] I have tested on a phone device/emulator
 - [x] I have tested on a tablet device/emulator
 - [x] I have tested on a kiosk device/emulator

#### Phone
<details>

</details>

#### Tablet
<details>

</details>

#### Kiosk
<details>

</details>

### CODE REVIEW

File: fabs/README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# FloatingActionButton component
+
+## FloatingActionButton design specs
+
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you just have to use the default `FloatingActionButton` component or the `VitaminFloatingActionButton*` component.
```
Comment (charlottemougenot) [inline fabs/README.md:9]: `VitaminFloatingActionButton*` The '*' is there on purpose ?

**Comment by charlottemougenot** [comment]
Could you add the fabs in project's readme list of artefacts ?

**Review by charlottemougenot — CHANGES_REQUESTED** [review]
**Disabled fab visible**

As you can see in this screenshot, we can see what I think is elevation or shade of the disabled fab. Is it possible to remove it to match specs ?

disabled_fab_elevation

File: fabs/README.md (hunk 1 above)
Comment (MaximeJineMaraval) [reply fabs/README.md:9]: Sniper ! Error from me, I will fix it

Comment (MaximeJineMaraval) [reply fabs/README.md:9]: Will be fixed in the next PR update

**Comment by MaximeJineMaraval** [comment]
> **Disabled fab visible**
> 
> As you can see in this screenshot, we can see what I think is elevation or shade of the disabled fab. Is it possible to remove it to match specs ?

Fix in progress

> Could you add the fabs in project's readme list of artefacts ?

Will be fixed in the next PR update

**Review by charlottemougenot — APPROVED** [review]

**Pull request merged**